### ISSUE TITLE
sitemap is empty when only app.vue format is used and the pages directory is non-existent

### ISSUE THREAD

**Issue by idorenyinudoh** [harlan-zw/nuxt-seo#255]
### Describe the bug

<img width="1680" alt="Screenshot 2024-06-19 at 15 31 48" src="https://github.com/harlan-zw/nuxt-seo/assets/44336070/13abf79e-8bf6-420b-ae3d-0a3694e328f2">
<img width="1680" alt="Screenshot 2024-06-19 at 15 32 37" src="https://github.com/harlan-zw/nuxt-seo/assets/44336070/6f250d76-5272-45fe-abaf-05fcd47fe6e6">

### System / Nuxt Info

```Shell
- Operating System: Darwin
- Node Version:     v18.18.0
- Nuxt Version:     3.11.2
- CLI Version:      3.11.1
- Nitro Version:    2.9.6
- Package Manager:  npm@10.8.1
- Builder:          -
- User Config:      devtools, modules, colorMode, runtimeConfig, site
- Runtime Modules:  nuxt-gtag@1.2.1, @nuxtjs/tailwindcss@6.12.0, @nuxtjs/color-mode@3.4.1, @nuxtjs/sitemap@5.2.0
- Build Modules:    -
```

**Comment by harlan-zw** [comment]
This is indeed a bug, it should ideally always have a root path. I'd consider why you'd want to use a sitemap for a site with a single page though.

**Comment by idorenyinudoh** [comment]
🤷🏽‍♂️

**Comment by harlan-zw** [comment]
This is fixed in v5.3.0 of the sitemap module, it will always append the base URL into the app sources.
